aeron-client/src/main/c/aeronc.h

```diff
@@ -34,6 +34,20 @@ typedef struct aeron_buffer_claim_stct aeron_buffer_claim_t;
 typedef struct aeron_publication_stct aeron_publication_t;
 typedef struct aeron_exclusive_publication_stct aeron_exclusive_publication_t;
 typedef struct aeron_header_stct aeron_header_t;
+typedef struct aeron_header_values_stct
```

Comment:
Should be specifying the packing details here if it is an overlay?

Reply:
Done.